python/simplesauce/session.py

```diff
@@ -0,0 +1,85 @@
+import os
+from selenium import webdriver
+from selenium.webdriver.remote.remote_connection import RemoteConnection
+from simplesauce.options import SauceOptions
+
+
+SAUCE_USERNAME = os.getenv('SAUCE_USERNAME', None)
+SAUCE_ACCCESS_KEY = os.getenv('SAUCE_ACCESS_KEY', None)
+
+us_ondemand = 'ondemand.us-west-1.saucelabs.com'
+eu_ondemand = 'ondemand.eu-central-1.saucelabs.com'
+
+US_SAUCE_DC_URL = 'https://{}:{}@{}/wd/hub'.format(SAUCE_USERNAME, SAUCE_ACCCESS_KEY, us_ondemand)
+EU_SAUCE_DC_URL = 'https://{}:{}@{}/wd/hub'.format(SAUCE_USERNAME, SAUCE_ACCCESS_KEY, eu_ondemand)
+
+
+class SauceSession():
+
+    def __init__(self, data_center='us', username=None, access_key=None, options=None):
+
+        # TODO: flesh this out
+        self.options = options if options else SauceOptions()
+
+        self._username = username if username else SAUCE_USERNAME
+        self._access_key = access_key if access_key else SAUCE_ACCCESS_KEY
+        self._data_center = data_center
+
+        if data_center.lower() == 'eu':
+            self.remote_url = EU_SAUCE_DC_URL
+        elif data_center.lower() == 'us':
+            self.remote_url = US_SAUCE_DC_URL
+        else:
+            raise KeyError("Invalid Data Center value, please select from 'us' or 'eu'")
```

Comment:
See comment below--we need to make sure we add the Headless data center, and keep this somewhat open for new endpoints. Could the "else" clause validate the data_center parameter as a URL, try to open a session at the URL as an endpoint, and then raise the KeyError if it doesn't work? Is that non-Pythonic?